src/main/resources/templates/fragments/item-modals.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+    <div th:fragment="addCollectionModal">
+        <div class="modal fade" id="addCollectionModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
+            <div class="modal-dialog modal-dialog-centered" role="document">
+                <div class="modal-content">
+                    <div class="modal-header">
```

Comment:
Small sample of the new "item-modals" page. Thymeleaf fragments are great for reusable bits of our code, and these modals seem like they most definitely will be reused if the website expands. It is best that we only need to edit the modals once in order to change them throughout the whole application.

Citation: https://www.thymeleaf.org/doc/articles/layouts.html